src/style.css

```diff
@@ -1 +1,36 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-content: space-between;
+  margin-left: 2px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  justify-content: center;
```

Comment:
The `justify-content` property is not applicable to individual flex items like `.stars__star`. It should be applied to the flex container instead. Consider removing it from here.